packages/fluentui/react-northstar/src/components/Form/FormField.tsx

```diff
@@ -112,14 +127,31 @@ const FormField: React.FC<WithAsProp<FormFieldProps>> & FluentComponentStaticPro
     }),
   });
 
-  const messageElement = Text.create(message, {
+  const messageElement = Text.create(errorMessage || message, {
     defaultProps: () => ({
       styles: resolvedStyles.message,
     }),
   });
 
+  const iconElement = Box.create(errorIndicator || satisfactoryIndicator, {
+    defaultProps: () =>
+      getA11yProps('icon', {
+        title: name,
+        styles: resolvedStyles.icon,
+      }),
+  });
+
   const controlElement = Box.create(control || {}, {
-    defaultProps: () => ({ required, id, name, type, styles: resolvedStyles.control }),
+    defaultProps: () =>
+      getA11yProps('control', {
+        required,
+        id,
+        name,
+        type,
+        icon: !!errorMessage ? iconElement : null,
+        ...(!!satisfactoryIndicator && { satisfactoryIndicator }),
```

Comment:
@layershifter @pompomon I had to add this condition to only pass it when defined otherwise it would break saying it's not a valid DOM property in the box creation for different inputs, e.g `Checkbox`. `icon` works only because it's not `camelCase`. Glad to change if there's a better alternative.